docker/sink-compose.yml

```diff
@@ -15,7 +15,10 @@
 version: '2.4'
 services:
   sink-server:
-    image: "hapiproject/hapi:latest"
+    # TODO: This is a temporary fix to rollback the newest version of
+    # HAPI which causes the e2e test to fail; uncomment when root cause fixed.
+    #image: "hapiproject/hapi:latest"
+    image: "hapiproject/hapi:latest-distroless"
```

Comment:
@omarismail94 FYI, it took me a while to find the root cause of an e2e test failure which was being triggered on this PR. It is not caused by the changes here but rather a new release of HAPI (which we were picking up here). The root cause seems to be related to [this MDM feature](https://hapifhir.io/hapi-fhir/docs/server_jpa_mdm/mdm_details.html) creating two patient resources when a patient is added. I will create a separate issue and PR to fix that problem. This is just a temporary fix as workaround (picking an older HAPI version).